EventListener/ContactSubscriber.php

```diff
@@ -43,18 +46,25 @@ class ContactSubscriber implements EventSubscriberInterface
      */
     private $configProvider;
 
+    /**
+     * @var CustomItemXrefContactRepository
+     */
+    private $customItemXrefContactRepository;
+
     public function __construct(
         EntityManager $entityManager,
         TranslatorInterface $translator,
         CustomItemRouteProvider $routeProvider,
         CustomItemModel $customItemModel,
-        ConfigProvider $configProvider
+        ConfigProvider $configProvider,
+        CustomItemXrefContactRepository $customItemXrefContactRepository
```

Comment:
The entity manager is already injected, do you think there is a need for repo to be injected as well?

Reply:
@ts-navghane as we discussed during the team sync, DI is better than getting the repo from the entity manager, so let's keep it as is.